client/packages/invoices/src/Prescriptions/LineEditView/PrescriptionLineEditForm.tsx

```diff
@@ -254,9 +254,11 @@ const dosesSummary = (
     0
   );
 
+  const roundedDoses = NumUtils.round(totalDoses);
+
   const unitWord = t('label.doses-plural', {
-    count: NumUtils.round(totalDoses),
+    count: roundedDoses,
```

Comment:
Feels like there's probably other places similar to this, outbound shipments for example? Other rounding needed?